firestore/query.go

```diff
@@ -389,11 +389,16 @@ func (q *Query) fieldValuesToCursorValues(fieldValues []interface{}) ([]*pb.Valu
 		if ord.isDocumentID() {
 			// TODO(jba): support DocumentRefs as well as strings.
 			// TODO(jba): error if document ref does not belong to the right collection.
-			docID, ok := fval.(string)
+			cursor, ok := fval.(*pb.Cursor)
 			if !ok {
-				return nil, fmt.Errorf("firestore: expected doc ID for DocumentID field, got %T", fval)
+				docID, ok := fval.(string)
+				if !ok {
+					return nil, fmt.Errorf("firestore: expected doc ID for DocumentID field, got %T", fval)
```

Comment:
I think "doc ID or cursor" should be written in the error message

Reply:
That isn't my error message, it was written by someone else before. I left it untouched.